File: Day1/Armstrong Number.cpp
#include<iostream>
using namespace std;
#include<math.h>

int main()
{
    int n,d=0,sum=0;
    cin>>n;
    int num = n;
    while(num!=0){
        d++;
        num/=10;
    }
    num=n;
    while(num!=0){
        sum += pow((num%10),d);
        num/=10;
    }
    if(n==sum)
        cout<<"Armstrong";
    else
        cout<<"Not Armstrong";

}

File: Question Bank/Armstrong Number.py
n = int(input())
num = n
d = 0
while num!=0:
    d+=1
    num/=10
    
sum = 0
num = n

while num!=0:
    sum += ((num%10)**d)
    num/=10
    print(sum)

if sum==n:
    print('{} is an Armstrong number'.format(n))
else:
    print('{} is not a Armstrong number'.format(n))